addon/components/tool-tipster.js

```diff
@@ -84,8 +84,8 @@ export default Ember.Component.extend({
       }
     });
     options.trigger = this.get('triggerEvent');
-    // Handle SafeString
-    if (content instanceof SafeString) {
+    // Handle htmlSafe
+    if (content instanceof htmlSafe) {
       options.content = content.toString();
```

Comment:
ok!  I can work on that.  This my very first PR ever! Sorry about applying 2 corrections under one PR.